zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -256,16 +243,10 @@ benchmarks! {
     }: { call.dispatch_bypass_filter(destroy_origin)? }
 
     admin_destroy_reported_market{
-        // a = total accounts
-        let a in 0..10;
-        // b = num. accounts with assets
-        let b in 0..10;
-        // c = num. asset types
-        let c in (T::MinCategories::get().into())..T::MaxCategories::get().into();
-        // Complexity: c*a + c*b ∈ O(a)
-
-        let c_u16 = c.saturated_into();
-        let (caller, market_id) = setup_resolve_common_categorical::<T>(a, b, c_u16)?;
+        let total_accounts = 1000;
+        let accounts_with_assets = 750;
+        let categories :u16 = T::MaxCategories::get().saturated_into();
+        let (caller, market_id) = setup_resolve_common_categorical::<T>(total_accounts, accounts_with_assets, categories)?;
```

Comment:
We don't need any outcome token holding account now, so we can remove the overhead of creating them in the benchmarks (which get executed thousands of times):
```suggestion
        let categories = T::MaxCategories::get();
        let (caller, market_id) = setup_resolve_common_categorical::<T>(0, 0, categories)?;
```

Reply:
I will take care of it in https://github.com/zeitgeistpm/zeitgeist/pull/812